package.json

```diff
@@ -62,6 +62,9 @@
     "sinon-chai": "^3.5.0",
     "svgo": "^1.3.2",
     "typescript": "^4.1.3",
+    "stylelint": "^14.1.0",
+    "stylelint-config-recommended-scss": "^5.0.2",
+    "postcss": "^8.3.3",
```

Comment:
Do we need to install `postcss` ? I see that [the shared configuration mentions bundling a few configurations](https://www.npmjs.com/package/stylelint-config-recommended-scss), including the PostCSS one. If we were to instead extend the other two specifically, could we drop the extra dependency?

Reply:
I ran into this issue: https://github.com/bjankord/stylelint-config-sass-guidelines/issues/203

It suggested adding postcss, and it did resolve it.

Reply:
I suppose that makes sense if they're using PostCSS to parse SASS 👍